recipes/pyxtensor-feedstock/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pyxtensor" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/{{ name }}/archive/{{ version }}.tar.gz
```

Small typo in the URL
```suggestion
  url: https://github.com/tdegeus/{{ name }}/archive/v{{ version }}.tar.gz
```